sys/include/net/ieee802154/radio.h

```diff
@@ -190,6 +194,17 @@ typedef enum {
      */
     IEEE802154_RADIO_INDICATION_RX_START,
 
+    /**
+     * @brief the transceiver received a frame with an invalid crc.
+     *
+     * The transceiver might not stay in @ref IEEE802154_TRX_STATE_RX_ON
+     * after receiving an invalid CRC. Therefore if the radio should keep
+     * listening for new frames the radio must go to
+     * @ref IEEE802154_TRX_STATE_TRXOFF or @ref IEEE802154_TRX_STATE_TXON
```

Comment:
Doxygen will fail here. It should point to IEEE802154_TRX_STATE_TRX_OFF and IEEE802154_TRX_STATE_TX_ON